app/client/src/components/pages/StateTribal.Tabs.AdvancedSearch.js

```diff
@@ -949,6 +978,7 @@ function AdvancedSearch() {
             // set the display back to the default option
```

Comment:
This was an issue before this PR, but I only saw it while testing the below addition: it seems like the intention of the `indexOfDisplay` is to keep a selection across searches if it's in both. However, the groups used in `react-select` result in a nested array and the `indexOfDisplay` is only ever found for "Overall Waterbody Condition". The above lines 975-977 could be changed to:
```
const indexOfDisplay = newDisplayOptions
  .flatMap((item) => item.options ?? item)
  .findIndex((item) => item.value === selectedDisplayOption.value);
```

Reply:
Good catch! I think that select used to not have the groupings.